### DISCUSSION TITLE
Are there any plans to release 2.18.11 any time soon?

### DISCUSSION THREAD

**Discussion by sergey-nilga** [COVESA/dlt-daemon#661] [General]
Hello people doing great job!
Are there any plans to release 2.18.11 any time soon?
It would be great to have all new features that you have added since last release become a part a next release!

**Comment by minminlittleshrimp** [comment]
Hello @michael-methner 
Sorry to bother you. Could we plan for next release?
Thank you

**Comment by marck-brusa** [comment]
Hello everyone, it's been a while since there was a release, and we have several issues with the 2.18.10 release which appear to be fixed in master. 

Any plans for a 2.18.11 release?

**Reply by minminlittleshrimp** [reply]
Dear,
protocol v2 is under development by Volvo Group, kindly check: https://github.com/COVESA/dlt-daemon/discussions/765

**Comment by minminlittleshrimp** [comment]
Hi @sergey-nilga 
Finally 2.18.11 has been released https://github.com/COVESA/dlt-daemon/releases/tag/v2.18.11
Thanks for being patient
